pkg/webhook/webhookreporter.go

```diff
@@ -68,9 +68,24 @@ func (r *WebhookReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		}
 
 	}
+
 	return nil
 }
 
+// SendWebhookReport sends a report directly via webhook without going through CRD reconciliation
+// This is used when AltReportStorageEnabled is true and reports are written to filesystem
+func SendWebhookReport(reportData []byte, config etc.Config, log logr.Logger) {
```

Comment:
I'd improve a name for this function.
something like `SendWebhookReportIfAvailable`, in favor of other functions in `config`: https://github.com/aquasecurity/trivy-operator/blob/main/pkg/plugins/trivy/config.go#L375

@simar7 WDYT?